src/main/java/com/toptrumps/online/api/request/PlayerState.java

```diff
@@ -1,28 +1,32 @@
 package com.toptrumps.online.api.request;
 
+import com.fasterxml.jackson.annotation.JsonInclude;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.toptrumps.core.card.Card;
 import com.toptrumps.core.player.AIPlayer;
 import com.toptrumps.core.player.Player;
 
+@JsonInclude(JsonInclude.Include.NON_NULL)
 public class PlayerState {
 
     private int id;
     private boolean isAIPlayer;
     private String name;
     private int deckCount;
     private Card topCard;
+    private int numberOfRoundWins;
```

Comment:
Do you want me to pass you the number of round winds every turn?

Reply:
Nopes. You're only passing me data AFTER the game has ended.